aesara/graph/opt.py

```diff
@@ -645,32 +648,32 @@ def process_node(self, fgraph, node):
                         )
                     )
 
-                # if the current node has assert input, it should not be
-                # replaced with a candidate node which has no assert input
-                elif node_has_assert and not cand_has_assert:
-                    pairs = list(
-                        zip(
-                            candidate.outputs,
-                            node.outputs,
-                            ["merge"] * len(node.outputs),
-                        )
-                    )
-                else:
-                    new_inputs = self.get_merged_assert_input(node, candidate)
-                    new_node = node.op(*new_inputs)
-                    pairs = list(
-                        zip(
-                            node.outputs,
-                            new_node.owner.outputs,
-                            ["new_node"] * len(node.outputs),
-                        )
-                    ) + list(
-                        zip(
-                            candidate.outputs,
-                            new_node.owner.outputs,
-                            ["new_node"] * len(node.outputs),
-                        )
-                    )
+                # # if the current node has assert input, it should not be
+                # # replaced with a candidate node which has no assert input
+                # elif node_has_assert and not cand_has_assert:
+                #     pairs = list(
+                #         zip(
+                #             candidate.outputs,
+                #             node.outputs,
+                #             ["merge"] * len(node.outputs),
+                #         )
+                #     )
+                # else:
+                #     new_inputs = self.get_merged_assert_input(node, candidate)
+                #     new_node = node.op(*new_inputs)
+                #     pairs = list(
+                #         zip(
+                #             node.outputs,
+                #             new_node.owner.outputs,
+                #             ["new_node"] * len(node.outputs),
+                #         )
+                #     ) + list(
+                #         zip(
+                #             candidate.outputs,
+                #             new_node.owner.outputs,
+                #             ["new_node"] * len(node.outputs),
+                #         )
+                #     )
```

Comment:
Why not remove this?

Reply:
We should consider re-enabling it some time, so I don't want to remove it yet.